pkg/store/storepb/custom.go

```diff
@@ -494,39 +495,43 @@ func (c *SeriesStatsCounter) CountSeries(seriesLabels []labelpb.ZLabel) {
 	}
 }
 
-func (c *SeriesStatsCounter) Count(series *Series) {
-	c.CountSeries(series.Labels)
-	for _, chk := range series.Chunks {
-		if chk.Raw != nil {
-			c.Chunks++
-			c.Samples += chk.Raw.XORNumSamples()
-		}
+func (c *SeriesStatsCounter) Count(r *SeriesResponse) {
```

Comment:
I'd prefer to introduce another function to record bytes fetched, but if you decide to change this function's interface, it is better to add unit tests